src/backend/mod.rs

```diff
@@ -85,4 +89,24 @@ impl DNSBackend {
 
         DNSResult::Success(results)
     }
+
+    // reverse lookup must return a single name resolved via mapping
+    pub fn reverse_lookup(&self, requester: &IpAddr, lookup_ip: &IpAddr) -> String {
```

Comment:
@mheon So one more thing i figured out is we must return `Vec<String>` cause currently list also contains `container id` and `name` so we might end up returning `containerid`, I have also made change in previous PR if you need any reference: https://github.com/containers/aardvark-dns/pull/39